[skip ci] Update INSTALLING.md, Verification steps moved to option 1

bbeggsTT opened the pull request:
You're All Set section included in Option 1: From Source section.

### Ticket
[PR#23602](https://github.com/tenstorrent/tt-metal/issues/23602)

### Problem description
You're All Set section only applied to Option 1: From Source

### What's changed
Moved You're All Set section into Option 1: Form Source section so that users don't think that You're All Set section applies to Option 2 and 3 as well. It only applies to Option 1.

### Checklist
- [ ] [All post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/all-post-commit-workflows.yaml) CI passes
- [ ] [Blackhole Post commit](https://github.com/tenstorrent/tt-metal/actions/workflows/blackhole-post-commit.yaml) CI with demo tests passes (if applicable)
- [ ] [Model regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-models.yaml) CI passes (if applicable)
- [ ] [Device performance regression](https://github.com/tenstorrent/tt-metal/actions/workflows/perf-device-models.yaml) CI passes (if applicable)
- [ ] (For models and ops writers) [Single-card demo tests](https://github.com/tenstorrent/tt-metal/actions/workflows/single-card-demo-tests.yaml) CI passes (if applicable) See [recommended dev flow](https://github.com/tenstorrent/tt-metal/blob/main/models/MODEL_ADD.md#a-recommended-dev-flow-on-github-for-adding-new-models).
- [ ] New/Existing tests provide coverage for changes

bbeggsTT committed:
[skip ci] Update INSTALLING.md, Verification steps moved to option 1

You're All Set section included in Option 1: From Source section.

dimitri-tenstorrent commented on INSTALLING.md:
````diff
@@ -210,36 +233,7 @@ To try our pre-built models in `models/`, you must:
   sudo apt-get install cpufrequtils
   sudo cpupower frequency-set -g performance
   ```
-
````
What should the wheel users do next? I am not sure it is within the scope of the same ticket.

Would you like them to try the Basic Examples in TTNN just like we are directing the Docker Release Image users?

bbeggsTT replied:
Each of the other sections has a suggestion to visit the correspoding repo. I think we can add a link to the tt-metal repo landing page to look at some models or tech reports.

bbeggsTT committed:
Update INSTALLING.md

Updated with link to tt-metal repo.

bbeggsTT committed:
Merge branch 'main' into bbeggsTT-patch-4

bbeggsTT committed:
Merge branch 'main' into bbeggsTT-patch-4

blozano-tt commented on INSTALLING.md:
````diff
@@ -192,7 +192,30 @@ source python_env/bin/activate
+
+  | Card             | Command                              |
+  |------------------|--------------------------------------|
+  | Grayskull        | ```export ARCH_NAME=grayskull```     |
````
```suggestion
```
There is no grayskull

blozano-tt commented on INSTALLING.md:
````diff
@@ -192,7 +192,30 @@ source python_env/bin/activate
+
+- Then, try running a programming example:
+  ```
+  python3 -m ttnn.examples.usage.run_op_on_device
````
This does not work. Examples are not bundled in the wheel.

See explanation as to why here:
```
In regards to our earlier discussion about embedding ttnn examples in the wheel.
I am obviously opposed to it, and I continue to feel that way after thinking about it a bit more.
When it comes to packaging things, structuring things, I like to not be creative, and follow what is industry standard, and to this point in my career, I have never seen or heard of any product bundling examples in python module site-packages dir. Something like that does not serve as a programming example.
Our ttnn product is framework-like, so we really should follow the example set by other frameworks.
https://github.com/pytorch/examples
https://github.com/tensorflow/examples
https://github.com/microsoft/onnxruntime-inference-examples
The above repos don't even house the examples with the product repo, so there is no chance they put examples in their wheel.
I think we should also put all examples / models in an external repo outside of tt-metal. Separation of concerns, break problems into component pieces.
```

blozano-tt reviewed (changes requested):
Left some objecting comments.

bbeggsTT committed:
Update INSTALLING.md

Edits from comments from BLozano:
- Grayskull entry removed.
- Non-functional programming example removed.

bbeggsTT committed:
Update INSTALLING.md

Merge cruft removed.

bbeggsTT committed:
Merge branch 'main' into bbeggsTT-patch-4

bbeggsTT commented:
Closing this PR. Many changes were being made to INSTALLING.md during these commits making the point of the change moot.
